packages/react/src/primitives/types/checkbox.ts

```diff
@@ -0,0 +1,18 @@
+import { InputProps } from './input';
+
+export interface CheckboxProps extends InputProps {
+  /**
```

Comment:
Please make children required, since that's the label text.

Reply:
Sounds good, though user could still get around that by using `{}`, `null` or `undefined`. I was wondering if there is a way to tighten up the children type checking but it seems that it will always `& React.ReactNode` :(

Reply:
Hmm, is this because we are using React.FC type?

Reply:
Yea, I think `React.FC<PropsType>` will enforce that

Reply:
It's OK if it's a ReactNode, since that will allow them to style the label how they want. I think the main thing thought is to encourage them to provide it, and not use `<CheckboxField/>`.